logic/StateManager.py

```diff
@@ -33,41 +33,51 @@ def load_state(self, path: str) -> None:
     def reset(self):
         """Reset everything"""
         self.game.situation_manager.reset()
-        self.player.stats.reset()
+        self.player_stats.reset()
         self.game.state_manager.load_initial_state()
         self.game.situation_manager.load_situation()
         self.game.game_screen.update_text()
 
     def set_state(self):
         """Sets the player state to the state that is loaded"""
-        cur_stats = self.player.stats
+        cur_stats = self.player_stats
         cur_stats.athletic_ability = self.state[0]['player']['stats']['athletic ability']
         cur_stats.charisma = self.state[0]['player']['stats']['charisma']
         cur_stats.smartness = self.state[0]['player']['stats']['smartness']
         cur_stats.wisdom = self.state[0]['player']['stats']['wisdom']
-        self.player.stats.update_text()
+        for x in self.state[0]['player']['stats']:
```

Comment:
what the hell is going on here?

Reply:
im loading the stats and i dont know how to make it with less code than that lol

Reply:
try harder